data/human/variants.txt

```diff
@@ -2613,6 +2613,92 @@ ship "Scout" "Scout (Speedy)"
 
 
 
+ship "Scrapper" "Scrapper (Minimal)"
+	outfits
+		"Energy Blaster"
+		
+		"nGVF-AA Fuel Cell"
+		"LP036a Battery Pack"
+		
+		"X1700 Ion Thruster"
+		"X1200 Ion Steering"
+		"Hyperdrive"
+
+
+
+ship "Scrapper" "Scrapper (Plasma Engine)"
+	outfits
+		"Twin Blaster"
+		
+		"nGVF-AA Fuel Cell"
+		"LP036a Battery Pack"
+		"D14-RN Shield Generator"
+		
+		"Chipmunk Plasma Thruster"
+		"Chipmunk Plasma Steering"
+		"Hyperdrive"
+
+
+
+ship "Scrapper" "Scrapper (Gatling)"
+	outfits
+		"Gatling Gun"
+		"Gatling Gun Ammo" 3000
+		
+		"nGVF-AA Fuel Cell"
+		"LP036a Battery Pack"
+		"D14-RN Shield Generator"
+		
+		"Chipmunk Plasma Thruster"
+		"Chipmunk Plasma Steering"
+		"Hyperdrive"
+
+
+
+ship "Scrapper" "Scrapper (Boarding)"
+	outfits
+		"Twin Blaster"
+		
+		"nGVF-AA Fuel Cell"
+		"LP036a Battery Pack"
+		"D14-RN Shield Generator"
+		"Laser Rifle" 4
```

Comment:
```suggestion
		"KP-6 Photovoltaic Panel" 6
		"LP036a Battery Pack"
		"Bunk Room"
		"Fragmentation Grenades" 8
		"Laser Rifle"
```

If this version is aimed at boarding, perhaps it should have a little more focus on being able to board effectively? Based on the combo of the plasma engines and the boarding name, I would expect this is either going after tiny ships on its own, or preferably following in the wake of bigger pirates and quietly trying to capture somewhat bigger ships while the bigger pirate is still engaged against other threats.

Note: I left a single laser rifle there on the assumption that this ship is all about agressively boarding... But the captain's going to want a weapon with better defense so that they can retreat when everyone else has been killed, right? Self preservation at its finest.